backend/src/controllers/comment.js

```diff
@@ -20,3 +20,17 @@ exports.getComment = function({ params: { cid }, user }, res) {
         res)
         .catch(errorHandler(res))
 }
+
+/* PUT updated comment body */
+exports.putComment = function ({ user, params, body }, res) {
+    body.userID = user.id
+    commentModel.putComment(params.cid, body)
+        .then(() => exports.getComment({ user, params }, res))
```

Comment:
new pattern is not doing this 🥇

Reply:
But I dunno.

Reply:
What is the pattern? As far as i can see the other put routes return the object e.g. putReviewLikes

Reply:
https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#useful-post-responses

Reply:
It occurs to me you mean the pattern of calling exports.getThing, rather than the REST pattern of returning the updated resource from a PUT